src/main/resources/META-INF/plugin.xml

```diff
@@ -29,7 +29,7 @@
     <depends>com.intellij.properties</depends>
     <depends>org.jetbrains.idea.maven</depends>
     <depends>com.intellij.gradle</depends>
-    <idea-version since-build="222" until-build="222.*"/>
+    <idea-version since-build="222"/>
```

Comment:
Should we preemptively block for anything past 223.* or wait to see if it breaks us?

Reply:
I was considering that as well. Given how unstable LSP integration with IntelliJ is, might be better to add it in and bump when available and working.